www/grpc/server.go

```diff
@@ -49,9 +49,14 @@ func (s *Server) StartServer() error {
 		sync:   s.sync,
 		logger: s.logger,
 	}
+	walletServer := &walletServer{
```

Comment:
Cover added lines (L52 to L54) with test

Reply:
Technically we can't cover these lines because we use bufconn to test gRPC APIs: https://stackoverflow.com/questions/42102496/testing-a-grpc-service